x/validator-vesting/client/cli/query.go

```diff
@@ -0,0 +1,81 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/codec"
+
+	"github.com/kava-labs/kava/x/validator-vesting/types"
+)
+
+// GetQueryCmd returns the cli query commands for the kavadist module
+func GetQueryCmd(queryRoute string, cdc *codec.Codec) *cobra.Command {
+	valVestingQueryCmd := &cobra.Command{
+		Use:   types.ModuleName,
```

Comment:
Can we update `types.ModuleName` to `vesting` so it matches the REST api? It looks like it won't conflict with "vesting" in the latest version of cosmos-sdk, but not 100% sure here.

Reply:
That would be a breaking change, so can't update it as part of the client release.

Reply:
Ah, my comment wasn't clear - I meant adding a new route variable to use instead of `types.ModuleName`

Reply:
resolved by https://github.com/Kava-Labs/kava/pull/586/commits/12a904214881a7ddf80ebd465c7f49674aaba68e